org.openscenegraph.android/jni/JNIosg.cpp

```diff
@@ -93,14 +113,33 @@ JNIEXPORT void JNICALL Java_org_openscenegraph_osg_core_Node_nativeSetMode(JNIEn
 	}
 }
 
+JNIEXPORT void JNICALL Java_org_openscenegraph_osg_core_Node_nativeSetPointSize(JNIEnv* env, jclass, jlong cptr, jfloat pt_size)
+{
+	osg::Node* node = reinterpret_cast<osg::Node*>(cptr);
+	osg::Point* point = new osg::Point();
+	point->setSize((float)pt_size);
+	if(node != NULL)
+		node->getOrCreateStateSet()->setAttributeAndModes(point, osg::StateAttribute::ON);
+}
+
+JNIEXPORT void JNICALL Java_org_openscenegraph_osg_core_Node_nativeSetLineWidth(JNIEnv* env, jclass, jlong cptr, jfloat line_size)
+{
+	osg::Node* node = reinterpret_cast<osg::Node*>(cptr);
+	osg::LineWidth* lw = new osg::LineWidth();
+	lw->setWidth((float)line_size);
+	if(node != NULL)
+		node->getOrCreateStateSet()->setAttributeAndModes(lw, osg::StateAttribute::ON);
+}
+
```

Comment:
Rafa, I agree. But messing with the StateSet wrapping is something I wanted to avoid for now - especially as I usually rather work with geometry, hence more often work with the node concept.